applogic/src/api/conversation_details_cubit.rs

```diff
@@ -130,13 +131,14 @@ impl ConversationDetailsCubitBase {
     /// The not yet sent message is immediately stored in the local store and then the message is
     /// send to the DS.
     pub async fn send_message(&self, message_text: String) -> anyhow::Result<()> {
-        let domain = self.context.store.user_name().domain();
-        let content = MimiContent::simple_markdown_message(domain, message_text);
+        let content = MimiContent::simple_markdown_message(message_text);
```

Comment:
Messages don't have a domain anymore. Is that alright?

Reply:
I don't think we've used the domain for anything so far. We can probably pull it from the sender if we really need it. Should be fine to leave out for now.